tests/test_itemized.py

```diff
@@ -425,3 +425,21 @@ def test_filters_sched_e(self):
             results = self._results(api.url_for(ScheduleEView, **{label: values[0]}))
             assert len(results) == 1
             assert results[0][column.key] == values[0]
+
+    def test_filters_sched_e_efile(self):
+        filters = [
+            ('image_number', ScheduleEEfile.image_number, ['123', '456']),
+            ('committee_id', ScheduleEEfile.committee_id, ['C01', 'C02']),
+            ('support_oppose_indicator', ScheduleEEfile.support_oppose_indicator, ['S', 'O']),
+            ('is_notice', ScheduleEEfile.is_notice, [True, False]),
+            ('filing_form', ScheduleEEfile.form_type, ['F24N', 'F3XN'])
+        ]
+        for label, column, values in filters:
+            print(label, column, values)
+            [
+                factories.ScheduleEEfileFactory(**{column.key: value})
+                for value in values
+            ]
```

Comment:
Nice setup here!  May not need the `print` statement at this point, but not an issue.